docs/en-us/2.0.0/user_doc/architecture/design.md

```diff
@@ -78,7 +78,7 @@ scheduling system
 * **UI**
 
   The front-end page of the system provides various visual operation interfaces of the system,See more
-  at<a href="/en-us/docs/2.0.0/user_doc/system-manual.html" target="_self"> System User Manual </a>section。
+  at [System User Manual](../guide/system-manual.md) section。
```

Comment:
And for this dead link, maybe we better to change it to `../guide/homepage.md` and the content change to `Introduction to Functions`
![image](https://user-images.githubusercontent.com/15820530/147432618-9e789d8f-f9ab-4f96-8667-909d983d2d50.png)